ts/Series/Pie/PieDataLabel.ts

```diff
@@ -388,6 +413,31 @@ namespace ColumnDataLabel {
                     distributionLength,
                     distributionLength / 5
                 );
+
+                // Uncomment this to visualize the boxes
+                /*
+                points.forEach((point): void => {
+                    const box = point.distributeBox;
+                    point.dlBox?.destroy();
+                    if (box?.pos) {
+                        point.dlBox = chart.renderer.rect(
+                            chart.plotLeft + this.center[0] + (
+                                halfIdx ?
+                                    -this.center[2] / 2 - 100 :
+                                    this.center[2] / 2
+                            ),
+                            chart.plotTop + box.pos,
+                            100,
+                            box.size
+                        )
+                            .attr({
+                                stroke: 'silver',
+                                'stroke-width': 1
+                            })
+                            .add();
+                    }
+                });
+                // */
```

Comment:
After uncommenting this block the boxes look a little bit strange to me, but the data labels are placed correctly.
<img width="426" alt="image" src="https://github.com/user-attachments/assets/cad9a761-0c09-4769-923c-e38c600d33c2">

Reply:
Yes I know... In some cases the boxes are spot on, in other cases like this they are shifted. I haven't found out why that is, but I know something is fishy with the box distribution. Otherwise the CHC label wouldn't fall outside the chart in the first place.